docs/hardware/custom_hardware.mdx

```diff
@@ -14,3 +14,70 @@ poetry run 01 --server
 You may need to set additional parameters via [flags](/software/flags) depending on your setup.
 
 To transmit audio commands to 01, send LMC audio chunks to the websocket defined by your server.
+
```

Comment:
We might change/add more types to the [LMC page](https://mintlify.com/docs/content/components/cards), so I think we should just link to https://docs.openinterpreter.com/protocols/lmc-messages, maybe with a big [<Card>](https://mintlify.com/docs/content/components/cards) to draw attention to it